ush/ufoeval/run_ufo_hofx_test.sh

```diff
@@ -9,18 +9,55 @@
 # - Produce an EVA YAML
 # - Run EVA
 #-------------------------------------------------------------
-#--------------- User modified options below -----------------
-obtype=$1
-cycle=${2:-2021080100}
+
+# ==============================================================================
+usage() {
+  set +x
+  echo
+  echo "Usage: $0 [-c cycle] [-x] [-s] [-h] instrument [cycle]"
+  echo
+  echo "  -c  cycle to run DEFAULT=2021080100"
+  echo "  -x  don't run eva DEFAULT=run eva"
+  echo "  -s  just produce eva stats plots DEFAULT=produce lots of plots"
+  echo "  -h  display this message and quit"
+  echo
+  exit 1
+}
+
+# ==============================================================================
+
+cycle=2021080100
+run_eva=YES
+eva_stats_only=NO
+ 
+while getopts "c:hsx" opt; do
+  case $opt in
+    c)
+      cycle=$OPTARG
+      ;;
+    x)
+      run_eva=NO 
+      ;;
+    s)
+      eva_stats_only=YES
+      ;;
+    h|\?|:)
+      usage
+      ;;
+  esac
+done
+shift $((OPTIND - 1))
 
 if [ $# -lt 1 ] || [ $# -gt 2 ]; then
    echo "Incorrect number of arguments"
-   echo "Usage:"
-   echo $0 instrument [cycle]
-   exit 1
+   usage 
 fi
 
-machine=hera
+obtype=$1
+
+#--------------- User modified options below -----------------
+
+machine=orion
```

Comment:
for machine and GDASApp can we do:
`machine=${machine:-orion}` and `GDASApp=${GDASApp:-/path/to/stuff}`? that way an external script can call this directly without needing to edit it? (thinking for CI)

Reply:
I made the changes.  Hope you like them.